bundle/edu.gemini.dataman.app/src/main/java/edu/gemini/dataman/osgi/Activator.java

```diff
@@ -4,80 +4,125 @@
 
 package edu.gemini.dataman.osgi;
 
-import edu.gemini.dataman.context.DatamanConfig;
-import edu.gemini.dataman.context.DatamanContext;
-import edu.gemini.dataman.raw.RawCopier;
-import edu.gemini.dirmon.DirListener;
-import edu.gemini.dirmon.DirLocation;
-import edu.gemini.util.security.principal.StaffPrincipal;
+//import edu.gemini.dataman.context.DatamanConfig;
+//import edu.gemini.dataman.context.DatamanContext;
+//import edu.gemini.dataman.raw.RawCopier;
+//import edu.gemini.dirmon.DirListener;
+//import edu.gemini.dirmon.DirLocation;
+//import edu.gemini.util.security.principal.StaffPrincipal;
```

Comment:
Don't worry too much about the `Activator`.   I've commented out all the old Data Manager setup but will be coming back to it in the next PR (or a near future PR).  Some bits will be deleted and others modified and in fact I'll probably rewrite it in Scala.